test/functionalTest/cases/715_forward_xauth_token_to_cprs/propagate_xauth_token_for_update_and_query_standard_ops.test

```diff
@@ -0,0 +1,216 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Yes, as I just responded to the previous comment :-)
New func-tests are in the oven.